dev-test/utilities.test.js

```diff
@@ -36,18 +36,18 @@ describe("block height offset", () => {
   });
 
   it("should return zero offset", async () => {
-    const result = await executeScript("get-block-offset");
-    expect(result).toBe(0);
+    const [zeroOffset] = await executeScript("get-block-offset");
+    expect(zeroOffset).toBe(0);
   });
 
   it("should update offset", async () => {
     const manager = await getServiceAddress();
-    const result = await executeScript("get-block-offset");
-    expect(result).toBe(0);
+    const [updateOffset] = await executeScript("get-block-offset");
```

Comment:
it should be `zeroOffset` here as well :)

Reply:
Changed :)